dashboard/src/components/myProgress/discover/DiscoverProjectsPage.vue

```diff
@@ -112,6 +112,13 @@ aria-label="search for projects to pin"></b-input>
             <span v-if="data.item.nameHtml" v-html="data.item.nameHtml"></span>
             <span v-else>{{ data.item.name }}
             </span>
+            <div v-if="isEmailEnabled>
+              <b-button variant="outline-primary"
```

Comment:
I think it would make sense to put the button the same row as the name; could use bootstrap's ``row`` with 2 ``col``; it would be also a little more consistent how the buttons are placed in other views